src/Sulu/Bundle/CoreBundle/DependencyInjection/SuluCoreExtension.php

```diff
@@ -33,39 +33,6 @@ class SuluCoreExtension extends Extension implements PrependExtensionInterface
      */
     public function prepend(ContainerBuilder $container)
     {
-        // process the configuration of SuluCoreExtension
-        $configs = $container->getExtensionConfig($this->getAlias());
-        $parameterBag = $container->getParameterBag();
-        $configs = $parameterBag->resolveValue($configs);
-        $config = $this->processConfiguration(new Configuration(), $configs);
-
-        if (isset($config['phpcr'])) {
-            $phpcrConfig = $config['phpcr'];
-
-            // TODO: Workaround for issue: https://github.com/doctrine/DoctrinePHPCRBundle/issues/178
-            if (!isset($phpcrConfig['backend']['check_login_on_server'])) {
-                $phpcrConfig['backend']['check_login_on_server'] = false;
-            }
-
-            foreach (array_keys($container->getExtensions()) as $name) {
-                $prependConfig = [];
-                switch ($name) {
-                    case 'doctrine_phpcr':
-                        $prependConfig = [
-                            'session' => $phpcrConfig,
-                            'odm' => [],
-                        ];
-                        break;
-                    case 'cmf_core':
-                        break;
-                }
-
-                if ($prependConfig) {
-                    $container->prependExtensionConfig($name, $prependConfig);
-                }
-            }
-        }
```

Comment:
It IS a BC break if somebody is configuring PHPCR in the code bundle
(rather than with `parameters.yml`).

So I wonder if we should maintain this configuration point, but I think
the change makes sense (i.e. all PHPCR stuff is configured in the
DocumentManagerBundle).

On Thu, Apr 14, 2016 at 02:31:47AM -0700, Daniel Rotter wrote:
>    In
>    [1]src/Sulu/Bundle/CoreBundle/DependencyInjection/SuluCoreExtension.php:
> 
>  > -                switch ($name) {
>  > -                    case 'doctrine_phpcr':
>  > -                        $prependConfig = [
>  > -                            'session' => $phpcrConfig,
>  > -                            'odm' => [],
>  > -                        ];
>  > -                        break;
>  > -                    case 'cmf_core':
>  > -                        break;
>  > -                }
>  > -
>  > -                if ($prependConfig) {
>  > -                    $container->prependExtensionConfig($name, $prependConfig);
>  > -                }
>  > -            }
>  > -        }
> 
>    I guess this has been moved to another bundle? That's not a real BC break,
>    at least not if all bundles are registered, right?
> 
>    —
>    You are receiving this because you authored the thread.
>    Reply to this email directly or [2]view it on GitHub
> 
>    Reverse link: [3]unknown
> 
> References
> 
>    Visible links
>    1. https://github.com/sulu/sulu/pull/2277#discussion_r59687731
>    2. https://github.com/sulu/sulu/pull/2277/files/542841adaeb64b3060c415c9e532c563daf4c328#r59687731
>    3. https://github.com/sulu/sulu/pull/2277/files/542841adaeb64b3060c415c9e532c563daf4c328#r59687731